code/modules/pda/PDA.dm

```diff
@@ -504,8 +504,3 @@ var/global/list/obj/item/device/pda/PDAs = list()
 /obj/item/device/pda/process()
 	if(current_app)
 		current_app.program_process()
-
-/obj/item/device/pda/hit_check(speed)

```

Comment:
There's literally no equivalent to this in the new system, but this doesn't break throwing the PDA at nanomobs: It just means that you have to throw it *at* them, not past them or below them.